docs/whats_new.rst

```diff
@@ -32,7 +33,7 @@ Current
 
 Changelog
 ~~~~~~~~~
-- Nothing yet
+- Fixed RANSAC to give consistent results with a fixed seed across different chunk sizes, by `Austin Hurst`_ and `Yorguin Mantilla`_ (`#43 <https://github.com/sappelhoff/pyprep/pull/43>`_)
```

Comment:
This should go into the "bug" section

```suggestion
- Nothing yet

Bug
~~~
- Fixed RANSAC to give consistent results with a fixed seed across different chunk sizes, by `Austin Hurst`_ and `Yorguin Mantilla`_ (`#43 <https://github.com/sappelhoff/pyprep/pull/43>`_)
```

Also, we'll probably have to solve a git conflict with your two PRs editing "whatsnew" (not an issue, just a heads up)